readme.txt

```diff
@@ -130,7 +130,8 @@ If you get stuck, you can ask for help in the Plugin Forum.
 
 = 8.8.0 - xxxx-xx-xx =
 * Add - Implemented the "Update all subscriptions payment methods" checkbox on My Account → Payment methods for UPE payment methods.
-* Add - Add support for the new Stripe Checkout Element on the shortcode checkout page. 
+* Add - Add support for the new Stripe Checkout Element on the shortcode checkout page.
```

Comment:
This change is just a blank space at the end of the line which is being removed.

Reply:
Weird that GH isn't showing the whitespace being removed!